### ISSUE TITLE
"Upcoming live" and "Actually live" videos are caught in subsequent downloads

### ISSUE THREAD

**Issue by deldesir** [iiab/calibre-web#202]
Regression / Known issue (introduced by PR #199)

"Upcoming live" and "Actually live" videos are caught in subsequent downloads as requested videos. They will remain as such until they are available, that means having a duration value.

![image](https://github.com/iiab/calibre-web/assets/16546989/39c2ca8a-bbcd-4e0c-acab-439f4af6ba65)

_Originally posted by @deldesir in https://github.com/iiab/calibre-web/issues/199#issuecomment-2196814647_

**Comment by holta** [comment]
> Regression / Known issue (introduced by PR #199)
>
> "Upcoming live" and "Actually live" videos are caught in subsequent downloads as requested videos. They will remain as such until they are available, that means having a duration value.

According to https://github.com/iiab/calibre-web/pull/199#issuecomment-2196814647 itself:
> A new PR should [address] this regression by automatically deleting these videos records from xklb-metadata.db

Thanks! Can this PR be pulled together relatively quickly?

**Comment by deldesir** [comment]
I also think task messages should be adjusted according to https://github.com/iiab/calibre-web/issues/188#issuecomment-2190399720

**Comment by holta** [comment]
> I also think task messages should be adjusted according to [#188 (comment)](https://github.com/iiab/calibre-web/issues/188#issuecomment-2190399720)

"Tasks" view definitely sounds like the right place for this now.  Please go ahead: (keep things as simple as possible!)

- Tell people in plain English telling (1) what's really happening, (2) what they can do about it.
- The English explanation can always be clarified (or made more vividly clear) later.

**Comment by holta** [comment]
@deldesir is PR #194 fully adequate, as merged 9 days ago?

- PR #194

(Or does its code need to be slightly revised — if it too is somewhat related to explaining download mechanics of "ostensibly live" videos to users/operators ?)

**Comment by deldesir** [comment]
The status messages should be more explicit to let the user know about the type of video being processed, I mean the `live_status` of the video.
